aws-xray/src/main/java/io/opentelemetry/contrib/awsxray/GetSamplingTargetsResponse.java

```diff
@@ -35,8 +35,8 @@ static GetSamplingTargetsResponse create(
     @JsonCreator
     static SamplingTargetDocument create(
         @JsonProperty("FixedRate") double fixedRate,
```

Comment:
I think it's safe to assume this will always be present, since when creating a sampling rule `fixedRate` is required: https://docs.aws.amazon.com/xray/latest/api/API_SamplingRule.html